src/test/java/com/googleinterns/zoomtube/servlets/LectureServletTest.java

```diff
@@ -105,15 +106,16 @@ public void doGet_emptyDatabase_shouldReturnNoLecture() throws IOException {
     when(response.getWriter()).thenReturn(writer);
 
     servlet.doGet(request, response);
-    
+
     String json = content.toString();
     assertThat(json).startsWith("[]");
   }
 
   @Test
   public void doGet_oneLectureInDatabase_shouldReturnOneLecture() throws IOException {
     when(request.getParameter(LINK_INPUT)).thenReturn(TEST_LINK);
-    datastoreService.put(servlet.createLectureEntity(request));
+    datastoreService.put(LectureUtil.createEntity(
+        /* lectureName= */ "", /* videoUrl= */ TEST_LINK, /* videoId= */ TEST_ID));
```

Comment:
sorry I didn't make it clear, it is only for input which is not stored in a variables. TEST_LINK and TEST_ID do not need it.

Reply:
Please read: Document ambiguous literal arguments at the call site from https://docs.google.com/document/d/1ZQz9VulpweKEbg28epLO_tZhCboRF1DtWLrVZZlUYBE/edit#

Reply:
fixed